### PR TITLE
CartItem should have product config

### PR DESCRIPTION
**Pull request by 4Giedrius**
@hostinger/devs

### CODE REVIEW

File: src/CartItem.php [hunk 1]
```diff
@@ -341,6 +341,7 @@ public function toArray()
         return array(
             '__class' => get_class($this),
             'id' => $this->getId(),
+            'product' => $this->getProduct()->getConfig(),
```
Comment (vaidas-lungis) [inline src/CartItem.php]: IMHO `product` param is little bit misleading 
maybe `product_config` or
```
    'product' => array ('config' => $this->getProduct()->getConfig())
```
@zygintas  whats your opinion?

Comment (zygintas) [inline src/CartItem.php]: whatever

Comment (fordnox) [inline src/CartItem.php]: @vaidas-lungis totaly aggre with you

Comment (4Giedrius) [inline src/CartItem.php]: Of course, makes sense!

**Comment by zygintas** [comment]
merge? @4Giedrius

**Comment by vaidas-lungis** [comment]
Donald Trump was here

**Comment by 4Giedrius** [comment]
giphy-1

**Pull request merged**